java/client/src/main/java/glide/api/commands/servermodules/FT.java

```diff
@@ -703,6 +703,116 @@ public static CompletableFuture<String> aliasupdate(
         return executeCommand(client, args, false);
     }
 
+    /**
+     * Parse a query and return information about how that query was parsed.
+     *
+     * @param client The client to execute the command.
+     * @param indexName The index name to search into.
+     * @param query The text query to search. It is the same as the query passed as an argument to
+     *     {@link FT#search(BaseClient, String, String)}.
+     * @return A <code>String</code> representing the execution plan.
+     * @example
+     *     <pre>{@code
+     * FT.explain(client, "myIndex", "@price:[0 10]").get();
+     * // the result can look like (the result is a string):
+     * //  Field {
+     * //    price
+     * //    0
+     * //    10
+     * // }
+     * }</pre>
+     */
+    public static CompletableFuture<String> explain(
+            @NonNull BaseClient client, @NonNull String indexName, @NonNull String query) {
+        var args = concatenateArrays(new GlideString[] {gs("FT.EXPLAIN"), gs(indexName), gs(query)});
+        return executeCommand(client, args, false)
+                .thenApply(result -> ((GlideString) result).toString());
```

Comment:
```suggestion
        return FT.<GlideString>executeCommand(client, args, false)
                .thenApply(GlideString::toString);
```
(I hope I did no mistake there)